src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -80,7 +81,13 @@ public enum PropertyType {
 		 * Indicates that the condition is in a form of <code>something will/be something</code>,
 		 * also possibly in the negated form
 		 */
-		WILL
+		WILL,
+
+		/**
+		 * Indicates that the condition is in a multiform of <code>something will/be something</code> and <code>something can something</code>
+		 * also possibly in the negated form
+		 */
+		CAN_WILL
```

Comment:
Not a fan of this. You should just call register twice for both types.

Reply:
I didnt know you could do that

Reply:
oh yep! definitely easier than this approach 🙂

Reply:
Should I revert everything I did in PropertyCondition? or just take out that enum value